elasticsearch/src/test/java/org/apache/calcite/adapter/elasticsearch/ElasticSearchAdapterTest.java

```diff
@@ -220,6 +220,33 @@ public void basic() {
                 "size: 3"));
   }
 
+  @Test public void testLike() {
```

Comment:
Pls add more edge cases (from top of my head):
* like ''
* like '%'
* not like ''
* not like '%'

Reply:
also when field is `null`:
```sql
not nullField like '%'
```